.travis.yml

```diff
@@ -5,6 +5,6 @@ env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
 rvm:
-  - 2.1
```

Comment:
We had the most recent two rubies here before, so I just replaced with the most recent two rubies.  If we want to support 3 versions back, that's okay too.